core/templates/dev/head/combined-tests.spec.ts

```diff
@@ -46,12 +46,14 @@ getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting()
 );
-// Then we find all the tests. Note that known failing files are exempted;
+// Then we find all the tests and controllers,
+// directives, services and factories files.
+// Note that known failing and useless files are exempted;
```

Comment:
Can you explain more what "useless files" means here?

Reply:
Done